source/includes/common-minio-tiering.rst

```diff
@@ -81,4 +81,45 @@ aliases have the necessary permissions for creating policies and users
 on the deployment. See :ref:`minio-users` and :ref:`MinIO Policy Based Access Control <minio-policy>` for more
 complete documentation on MinIO users and policies respectively.
 
-.. end-create-transition-user-desc
\ No newline at end of file
+.. end-create-transition-user-desc
+
+.. start-transition-metadata-desc
+
+MinIO creates metadata for each transitioned object that identifies it's 
+location on the remote storage. This metadata is *required* for accessing the
+object. Applications cannot directly access the remote backend and distinguish
+a transitioned object independent of MinIO. Modification or loss of a 
+transitioned object's metadata file can result in the remote data becoming
+"disconnected" from the source. 
+
+.. end-transition-metadata-desc
+
+.. start-transition-bucket-access-desc
+
+MinIO *requires* exclusive access to the data on the remote storage tier. Object
+retrieval assumes no external mutation, migration, or deletion of stored
+objects. MinIO strongly recommends specifying a prefix on the remote bucket to
+better isolate and protect transitioned data.
```

Comment:
There is also the assumption that the remote storage tier does not have any lifecycle rules enabled for tiering or expiry on the remote bucket. This should be documented as well

Reply:
Let me know if the updated response looks better.